cfgov/wellbeing/urls.py

```diff
@@ -1,5 +1,6 @@
-from django.views.generic import TemplateView
+from django.conf.urls import url
 
+from core.views import TranslatedTemplateView
 from wellbeing.views import ResultsView
```

Comment:
We'll need to import `re_path` in Django 2 or import `url` as `re_path` in Django 1.11.

```suggestion

from core.views import TranslatedTemplateView
from wellbeing.views import ResultsView


try:
    from django.urls import re_path
except ImportError:
    from django.conf.urls import url as re_path
```